changes/1101.misc.rst

```diff
@@ -0,0 +1 @@
+Added `--template-branch` option to the `new` command for selecting a branch.
```

Comment:
This should be phrased in the form of a release note - i.e., from the perspective of an existing user being told about new features/fixes, rather than as a commit describing what has changed.
```suggestion
The `new` command now allows for specifying a custom template branch, as well as a custom template.
```

Also, the file should be named `.feature`, not `.misc`. This isn't a minor cleanup or administrative fix - it's a significant new feature!